app/javascript/packages/document-capture/components/document-side-acuant-capture.tsx

```diff
@@ -1,24 +1,25 @@
+import { useContext } from 'react';
 import { t } from '@18f/identity-i18n';
 import { FormError, FormStepsContext } from '@18f/identity-form-steps';
-import { useContext } from 'react';
+import type {
+  FormStepError,
+  OnErrorCallback,
+  RegisterFieldCallback,
+} from '@18f/identity-form-steps';
 import AcuantCapture from './acuant-capture';
 
-/** @typedef {import('@18f/identity-form-steps').FormStepError<*>} FormStepError */
-/** @typedef {import('@18f/identity-form-steps').RegisterFieldCallback} RegisterFieldCallback */
-/** @typedef {import('@18f/identity-form-steps').OnErrorCallback} OnErrorCallback */
-
-/**
- * @typedef DocumentSideAcuantCaptureProps
- *
- * @prop {'front'|'back'|'selfie'} side
- * @prop {RegisterFieldCallback} registerField
- * @prop {Blob|string|null|undefined} value
- * @prop {(nextValues:{[key:string]: Blob|string|null|undefined})=>void} onChange Update values,
- * merging with existing values.
- * @prop {FormStepError[]} errors
- * @prop {OnErrorCallback} onError
- * @prop {string=} className
- */
+interface DocumentSideAcuantCaptureProps {
+  side: 'front' | 'back' | 'selfie';
+  registerField: RegisterFieldCallback;
+  value: Blob | string | null | undefined;
+  /**
+   * onChange: Update values, merging with existing values.
```

Comment:
JSDoc makes it hard to visually distinguish, but `onChange` wasn't part of the previous description if we wanted to leave it out in the conversion.

```suggestion
   * Update values, merging with existing values.
```